src/Shared/Styles.elm

```diff
@@ -0,0 +1,16 @@
+module Shared.Styles exposing (..)
```

Comment:
ughh... styles module on a different file... is this person coming from Elm? 🤔

Reply:
Different approach than we are used to 🤷 Not necessarily bad part

Reply:
I actually think it's good they're trying to keep the style consistent.